formatters/jdt-core/build.properties

```diff
@@ -1,3 +1,20 @@
+#
+# Copyright 2010-2016. All work is copyrighted to their respective
```

Comment:
The concern is that the Apache license requires propagation of this information in the NOTICE file when downstream developers use AL2-licensed source code. It's easier for them to satisfy that requirement if the upstream communities using the AL2 license also provide this information in the NOTICE file rather than in the license headers.

The Apache Software Foundation (of which I'm a member) is particularly concerned about this for their projects, and while this is not an Apache project, and does not have to comply, the ASF does make it standard practice to request upstream users to move such notices to the NOTICE file. I'm just trying to get ahead of the game, in case any of this project's code gets used in an ASF project and we receive such a request.

I've also made it standard practice in my personal projects to follow this convention, if for no other reason than to play nice with downstream communities, such as those within the ASF.

In any case, it's a minor thing... and it could be done in a separate issue at some future point. I just thought I'd mention it here, since you were updating some of them.